src/shared/components/ActionBar/ActionBar.tsx

```diff
@@ -54,6 +54,13 @@ export const ActionBar: React.FC<ActionBarProps> = ({
   onCancelClick,
   primaryButtonTooltipText,
 }) => {
+  const renderButton = () =>
+    primaryButtonText ? (
+      <Button disabled={disabled} onClick={onConfirmClick} size="large" type="submit">
+        {primaryButtonText}
+      </Button>
+    ) : null
```

Comment:
Do we need this to be a function? I think this could just be a variable instead?